.travis.yml

```diff
@@ -10,6 +10,8 @@ matrix:
   - jdk: openjdk7
   - jdk: openjdk8
   - jdk: oraclejdk8
+  - jdk: openjdk11
```

Comment:
why not 9 and 10 too?

Reply:
Because 7 8 9.

Reply:
See #68 - I asked the same question but basically because 9/10 are not in LTS so can't be obtained on travis easily.